src/Aydsko.iRacingData/Leagues/CarClass.cs

```diff
@@ -0,0 +1,16 @@
+﻿// © 2022 Adrian Clark
+// This file is licensed to you under the MIT license.
+
+namespace Aydsko.iRacingData.Leagues;
+
+public class CarClass
```

Comment:
This `CarClass` is a subset of the properties defined in `Common.CarClass` but this endpoint doesn't provide a `relative_speed` nor `short_name` so I created another version to be honest with what the API actually provides.

Reply:
Yep, found several places where iRacing aren't consistent in what they provide. Makes things difficult.